community/kernel/src/main/java/org/neo4j/kernel/impl/store/CommonAbstractStore.java

```diff
@@ -131,7 +121,7 @@ public CommonAbstractStore(
                     e.addSuppressed( failureToClose );
                 }
             }
-            closeFileChannel();
+            close();
```

Comment:
Here you are closing `storeFile` twice one time before calling `close` and the second time in `close()`. It smells a little bit I think it should be cleaned up.

Moreover this looks like a change of the current behavior in case of failure when loading the store. Do we have enough test coverage to ensure that we are not messing up?